migtests/lib/yb.py

```diff
@@ -176,7 +176,17 @@ def fetch_all_function_names(self, schema_name="public") -> List[str]:
 		cur.execute(f"SELECT routine_name FROM information_schema.routines WHERE routine_schema = '{schema_name}'")
 		return [function[0] for function in cur.fetchall()]
 
-	def fetch_all_table_rows(self, table_name, schema_name="public") -> set[str]:
+	def fetch_all_table_rows(self, table_name, schema_name="public") -> any:
```

Comment:
Specifying `any` is as good as specifying nothing about the return value.
What made you replace the `set[str]` to `any`?